index.ts

```diff
@@ -27,22 +25,20 @@ export const setupPlugin: AvoInspectorPlugin['setupPlugin'] = async ({ config, g
     }
 }
 
-export const onEvent: AvoInspectorPlugin['onEvent'] = async (event, { config, global }) => {
+export const composeWebhook: AvoInspectorPlugin['onEvent'] = async (event, { config, global }) => {
```

Comment:
composeWebhook is non-async
```suggestion
export const composeWebhook: AvoInspectorPlugin['onEvent'] = (event, { config, global }) => {
```